### ISSUE TITLE
Team update - July 7, 2022

### ISSUE THREAD

**Issue by isabela-pf** [Quansight-Labs/jupyter-a11y-mgmt#142]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**

**Updates from last week :heavy_check_mark:**
- #133. 
     - It may be my mistake (or looking in limited time), but I'm missing the contrast feedback in the axe-core test results we ran. I can work without it, but I was hoping to give feedback on the results' navigability as I tried to use them.
     - I've turned the list of potential guidelines from last week into a [theming review checklist](https://github.com/isabela-pf/accessibility/pull/1). This is what I plan to use to review the current state of JupyterLab's theme and our own changes. I expect to make changes to it as we work with it more.
     - I've begun reviewing JupyterLab's default with this, but I'm not done yet.
- User testing outreach is on the roll. We are just starting this and templating the methods so we are consistent across participants and rounds. Tentative dates for testing are in August. 

**Challenges I faced and things I'd like assistance with 🙏**
- I don't know. Thanks for asking.

**My availability for next week**
- I'll here all week.

**Important items for discussion 💬**
- I don't think I have anything urgent.

**Comment by trallard** [comment]
- @trallard  will be syncing with Berkley folks to set up a follow-up call re accessibility efforts	
- @tonyfast  has the node packages working for the style theming 🤝  PR incoming

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- Thank you all for bearing with me during this blackhole of the website launch  🙏. The dust will settle in the next week.

**Updates from last week :heavy_check_mark:**
- I did this after yesterday's check-in; however, I submitted an issue about [upstreaming my fork to expect-axe-playwright](https://github.com/Widen/expect-axe-playwright/issues/21). 

**Challenges I faced and things I'd like assistance with 🙏**
Isabela and I had two pair review sessions for pull requests I submitted to fix various accessibility issues on the Labs and Consulting websites: 

- https://github.com/Quansight/Quansight-website/pull/288
- https://github.com/Quansight/Quansight-website/pull/294.
 
When we were discussing headings versus lists, we realized that we were hitting up against the limits of our own knowledge. I think we both feel that we would like to better understand how people use screen readers like NVDA. And not just what's in the docs, but how people actually use it in real life. It would be great if we both felt more confident that we could somewhat accurately mimic the ways that blind people commonly use a screen reader like NVDA. Not to imply of course that all people use it the same way or that all screen readers are the same. Anyway, I'm not sure what the solution is other than spending lots of time with it, but maybe there are also online courses. I haven't looked into it yet.

**My availability for next week**
- I am here all next week.
